validator/src/lib.rs

```diff
@@ -5,5 +5,5 @@ pub mod aggregation;
 mod jail;
 mod r#macro;
```

Comment:
Is this a typo?

Reply:
Not a typo. 

It is a raw identifier, as `macro` is a keyword. We could rename the file to not having to use it here, but it was not part of this PR.